addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
I may be missing something!  The purpose of this method is just for conveniently pushing arbitary objects into the infinityModel from the route, right @asross ?